target/linux/realtek/files-5.15/drivers/net/dsa/rtl83xx/debugfs.c

```diff
@@ -726,4 +829,7 @@ void rtl930x_dbgfs_init(struct rtl838x_switch_priv *priv)
 	debugfs_create_file("drop_counters", 0400, dbg_dir, priv, &drop_counter_fops);
 
 	debugfs_create_file("l2_table", 0400, dbg_dir, priv, &l2_table_fops);
+
+	debugfs_create_file("reg_rw", 0x200, dbg_dir, priv, &reg_rw_fops);
+	debugfs_create_file("mmd_rw", 0x200, dbg_dir, priv, &mmd_rw_fops);
```

Comment:
Given that `/dev/mem` exists for register access, and [`phytool`](https://github.com/wkz/phytool) could be (packaged and) used for MMD access, what makes this worthwhile to merge? You're free to keep this patch around, but I'm reluctant to merge stuff like this if there are equivalent tools available.

Reply:
dropped :+1: